pkg/kubectl/cmd/convert/convert.go

```diff
@@ -60,8 +60,8 @@ var (
 		kubectl convert -f . | kubectl create -f -`))
 )
 
-// ConvertOptions have the data required to perform the convert operation
-type ConvertOptions struct {
```

Comment:
I know golint doesn't like this, but just about every kubectl command names its options this way (`<something>Options`):

```
~/go/src/k8s.io/kubernetes/staging/src/k8s.io/kubectl/pkg/cmd (master) $ grep -re "type .*Options struct" | wc -l
71
~/go/src/k8s.io/kubernetes/staging/src/k8s.io/kubectl/pkg/cmd (master) $ grep -re "type Options struct" | wc -l
1
```

I'm not saying it is wrong to fix the lint warnings, but my concern is that instead of having a single consistent way of naming options (that golint happens not to like) that we end up with inconsistent naming, unless we undertake a larger effort to rename all the option structs.

Reply:
Ok, it makes sense. However it isn't possible to exclude only this name from golinter.
Should we then keep this file on `hack/.golint_failures` and maybe put a comment there saying to keep this file and the others with Options `<something>Options` without for now?

Reply:
Yeah, I guess golint doesn't have a way to override except at the file level. I'm OK with it the change, but I just sort of feel like we should follow through and change all the options if we change it in one. Maybe I'm thinking too much into it. I wonder if anyone else has any opinions on this...